drivers/dac/ltc2688/ltc2688.c

```diff
@@ -469,6 +469,8 @@ int32_t ltc2688_init(struct ltc2688_dev **device,
 		printf("LTC2688 successfully initialized\n");
 	else
 		printf("LTC2688 initialization error (%d)\n", ret);
+	
+	return ret;
```

Comment:
Put this ret inside if (!ret)
On error dev must be released.

Thanks for the fix.